### ISSUE TITLE
$PATH is empty when it's not in the parent's environment, unlike other shells

### ISSUE THREAD

**Issue by o11c** [oilshell/oil#743]
At the very least, `PATH` needs to be set.

**Comment by andychu** [comment]
It starts fine with `env -i` for me, but I see this difference:

```
$ env -i osh -c 'echo $PATH'

$ env -i bash -c 'echo $PATH'
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin

$ env -i dash -c 'echo $PATH'
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

I"m not actually sure where that $PATH comes from?

**Comment by andychu** [comment]
It looks like the default $PATH is a build-time configuration option... I guess every shell does it that way?

That's possible to do in Oil. 

https://askubuntu.com/questions/386629/what-are-the-default-path-values

 Ubuntu has `/etc/environment`, but that seems to be just a "nice to know", not the authoritative source.

**Comment by andychu** [comment]
I guess the more salient question is: what's the default default?

If you just do `./configure`, what is the default `$PATH` set to?

**Comment by andychu** [comment]
Fixed in 0.9.7: https://www.oilshell.org/blog/2022/01/notes-themes.html
